src/libData/AccountData/AccountStore.cpp

```diff
@@ -335,6 +339,20 @@ bool AccountStore::UpdateCoinbaseTemp(const Address& rewardee,
     //Should the nonce increase ??
 }
 
+boost::multiprecision::uint256_t
+AccountStore::GetNonceTemp(const Address& address)
+{
+    auto it = m_accountStoreTemp->GetAddressToAccount()->find(address);
+    if (it != m_accountStoreTemp->GetAddressToAccount()->end())
```

Comment:
Directly compare find()/end(), to reduce the usage of extra variable "it"

Reply:
Noted